src/Generator.js

```diff
@@ -72,6 +78,20 @@ const getNewPhase = (line) => {
   return null;
 };
 
+const getFeatureFileLanguage = (featureFilename, fileLines) => {
+  if (fileLines.length === 0) {
+    return language;
+  }
+  const givenLang = /^\s*#\s*language:\s*(\w+)\s*$/.exec(fileLines[0]);
```

Comment:
exec can cause interesting problems if the same line in-memory regex is re-called more than once, prefer `matches`